### DISCUSSION TITLE
Quest system is no longer compatible with Version 1.0

### DISCUSSION THREAD

**Discussion by gogrady4117** [XelaNull/7DTD-WhiteRiver_ToC#6] [General]
Quest system is no longer compatible with Version 1.0

**Comment by PaloAlo** [comment]
any plans to update for v1.0? if not i can put a fork or PR on git if you would prefer. ive been reviewing everything and so far it loads no errors, but there's always those little things that dont work properly until tested.  your code has a GNU license, but my only interest in this is to bring the mod to v1.0 for the sake of the modding community.  anyway, let me know what you'd like to do.

if you do decide to fork or PR then you will need to create another empty branch (afaik).

**Comment by XelaNull** [comment]
I haven't touched the code in awhile. If you have working code please generate pull request and I'll look at merging to master.

**Comment by XelaNull** [comment]
I'm slowing working towards fixing this modlet pack along with my many others. If you have a pull request ready, I'd appreciate it to help cut down on some of the work. But if you don't have one or can't provide it before I get to updating it, I'll just do the needful without.

**Comment by PaloAlo** [comment]
I do have most of it working. Arkangel and I worked on it for a week and
got almost all the modlets working.

There's about 5 things I wanted to look at since I'm not sure they worked
in the original version, or at least not worked in the rebuilt version that
JP did.

If you have a separate branch setup so I can push a PR I will send all the
known working ones.
If you want some help with the last bit we can collaborate on that as well
or you can finish it.

I'm very busy during the day but sometimes have an hour or two in the
morning or evening to work on stuff, so will send when I have time.

**Comment by PaloAlo** [comment]
you will need to make me a collaborator (i believe) before i can send the PR. 
i tried but it failed.

**Comment by XelaNull** [comment]
I've added you as a collaborator (check invite) and there is already a branch created to push changes into:

https://github.com/XelaNull/7DTD-WhiteRiver_ToC/tree/4-quest-system-is-no-longer-compatible-with-version-10

**Comment by XelaNull** [comment]
I've closed the Issue and converted to a Discussion, as that seemed more appropriate. We obviously all acknowledge the code needs updated for 1.0 and are working towards that. I'll pause making any more changes until you've had a chance to either create pull request or push into branch the changes you have so far. Please let me know what you are aware is outstanding beyond this point.

So far, I've only looked at the REQUIRED base modlet (but dont yet have it working). And I've updated all ModInfo.xml for new version and formatting.

**Comment by XelaNull** [comment]
Please make sure you edit the README on the git repo to add your names to the ACKWNOWLEDGEMENTS section. I'm huge proponent of always making sure credit is mentioned where it is due and have always made point within each modlet to credit anyone who contributes. 

And THANK YOU! I really appreciate you being willing to contribute and very happy to work with you to sort out whatever is remaining.

**Comment by gogrady4117** [comment]
-Arkangel here-
As PaloAlo stated we worked on updating the code for about a week which concluded a little over a week ago. I've been wrapped up with work for the last 6 days but did some light testing after completion. I have some errors showing for the steel ammo but I am not 100% sure that we implemented the updated code for our packaging during my testing. I will be spending the better part of the next eight days or so testing these mods individually and with others for compatibility and will update both here and Palo in discord. 

As an aside I want to extend my thanks to both you ZelaNull for creating it and Palo for heavy lifting to get this back up and running.  This is by far my #1 mod for this game and getting back on for 1.0 has been both a learning experience and a pleasure. Thank you both again for all your hard work and collaboration on this project.

**Comment by XelaNull** [comment]
In case anyone stumbles in here later.. this is all fixed up!
